app/scripts/views/mixins/save-options-mixin.js

```diff
@@ -85,14 +85,14 @@ const Mixin = {
     this.$('.error').addClass('hidden');
     this.$('.modal-success').removeClass('hidden');
     this.$('.modal-success').addClass('success');
-    this.$('.modal-success').html(msg);
+    this.$('.modal-success').html(this.translate(msg));
```

Comment:
You know, using .html like this is generally not considered good hygiene unless we escape `msg` or know what we are getting into.  @vbudhram - is any HTML written here or is it just text?